pkgs/development/python-modules/python3-application/default.nix

```
@@ -1,27 +1,18 @@
-{ stdenv, lib, isPy3k, buildPythonPackage, fetchFromGitHub, fetchpatch, zope_interface, twisted }:
+{ stdenv, lib, isPy3k, buildPythonPackage, fetchFromGitHub, zope_interface, twisted }:
 
 buildPythonPackage rec {
   pname = "python3-application";
-  version = "3.0.3";
+  version = "3.0.4";
```

3.0.4 is not released.

```suggestion
  version = "unstable-20220729";
```

---

@fabaff it has been released but not yet tagged on GitHub, for unknown reasons. It has been released on Debian unstable as 3.0.4.

---

https://github.com/AGProjects/python3-application/blob/master/debian/changelog for reference

---

https://github.com/AGProjects/python3-application/issues/4